src/subgraph/actions/listAllSales.ts

```diff
@@ -26,14 +26,18 @@ export const listAllSales = async <T>(
     }
 
     const queriedSales = queryResult.data.tokenSales.map((e) => {
-      return {
+      const sale: TokenSale = {
         timestamp: e.timestamp,
+        bidNonce: e.bidNonce,
         tokenId: e.tokenId,
         contract: e.contractAddress,
         saleAmount: e.amount,
         seller: e.seller.id,
         buyer: e.buyer.id,
-      } as TokenSale;
+        paymentToken: e.paymentToken ?? "",
```

Comment:
Can we instead of defaulting to `""` default to WILD token? Would make using this data far easier